matrix_io/vision/v1/vision.proto

```diff
@@ -55,24 +55,25 @@ message Rectangle {
 }
 
 enum EnumFacialRecognitionTag {
+  UNKNOWN = 0;
```

Comment:
It's important to note that any protobufs we previously stored (ie. VisionResults used for comparisons) will break, we will need to export them now to JSON / YAML so we can convert back to proto after merging this PR. This is probably relevant to @campos537 .